sdcm/cluster.py

```diff
@@ -2055,7 +2055,9 @@ def __init__(self, ec2_ami_id, ec2_subnet_id, ec2_security_group_ids,
         shortid = str(cluster_uuid)[:8]
         name = '%s-%s' % (cluster_prefix, shortid)
         user_data = ('--clustername %s '
-                     '--totalnodes %s' % (name, n_nodes))
+                     '--totalnodes %s '
+                     '--reflector http://reflector.scylladb.com/reflector.php'
```

Comment:
It means the parameter is not necessary and if you do not specify it the ami will use       http://reflector.scylladb.com/reflector.php.

The parameter is useful in case we are changing the reflector service and have an on the side instance and want to test the ami with it

There is no reason a user should use this param